src/plugins/cloudinary-analytics/index.js

```diff
@@ -16,7 +20,13 @@ class CloudinaryAnalytics {
     this.player.on(PLAYER_EVENT.SOURCE_CHANGED, () => {
       const metadata = this.getMetadata();
       if (metadata.cloudName && metadata.publicId) {
-        this.cloudinaryAnalytics.startManuallyNewVideoTracking(metadata);
+        this.currentVideMetadata = metadata;
+        this.cloudinaryAnalytics.startManualTracking(metadata, {
+          videoPlayerType: 'cloudinary video player',
+          videoPlayerVersion: VERSION
+        });
+      } else if (this.currentVideMetadata.cloudName !== metadata.cloudName || this.currentVideMetadata.publicId !== metadata.publicId) {
```

Comment:
what is this condition for? i thought we are providing a fallback if the customer is using custom urls and not stopping the sending of data, or am i missing something ?